modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/videoembed/plugin.js

```diff
@@ -0,0 +1,638 @@
+
+				if (!styleCheck && style.vendorPrefixes) {
+					styleCheck = style.vendorPrefixes.some(
+						vendorPrefix =>
+							embed.getStyle(vendorPrefix + style.name) ===
+							style.value
+					);
+				}
+
+				return styleCheck;
+			});
+
+			embedAlignment = centeredEmbed ? EMBED_ALIGNMENT.CENTER : null;
+		}
+
+		return embedAlignment;
+	};
+
+	/**
+	 * Removes the alignment value of an embed
+	 *
+	 * @param {CKEDITOR.dom.element} embed The embed element
+	 * @param {String} embedAlignment The embed alignment value to be removed
+	 */
+
+	const removeEmbedAlignment = function(embed, embedAlignment) {
+		if (
+			embedAlignment === EMBED_ALIGNMENT.LEFT ||
+			embedAlignment === EMBED_ALIGNMENT.RIGHT
+		) {
+			embed.removeStyle('float');
+
+			if (embedAlignment === getEmbedAlignment(embed)) {
+				embed.removeAttribute('align');
+			}
+		}
+		else if (embedAlignment === EMBED_ALIGNMENT.CENTER) {
+			CENTERED_EMBED_STYLE.forEach(style => {
+				embed.removeStyle(style.name);
+
+				if (style.vendorPrefixes) {
+					style.vendorPrefixes.forEach(vendorPrefix =>
+						embed.removeStyle(vendorPrefix + style.name)
+					);
+				}
+			});
+		}
+	};
+
+	/**
+	 * Sets the alignment value of an embed
+	 *
+	 * @param {CKEDITOR.dom.element} embed The embed element
+	 * @param {String} embedAlignment The embed alignment value to be set
+	 */
+
+	const setEmbedAlignment = function(embed, embedAlignment) {
+		removeEmbedAlignment(embed, getEmbedAlignment(embed));
+
+		if (
+			embedAlignment === EMBED_ALIGNMENT.LEFT ||
+			embedAlignment === EMBED_ALIGNMENT.RIGHT
+		) {
+			embed.setStyle('float', embedAlignment);
+		}
+		else if (embedAlignment === EMBED_ALIGNMENT.CENTER) {
+			CENTERED_EMBED_STYLE.forEach(style => {
+				embed.setStyle(style.name, style.value);
+
+				if (style.vendorPrefixes) {
+					style.vendorPrefixes.forEach(vendorPrefix =>
+						embed.setStyle(vendorPrefix + style.name, style.value)
+					);
+				}
+			});
+		}
+	};
+
+	const selectWidget = function(editor) {
+		setTimeout(() => {
+			const selection = editor.getSelection();
+
+			if (selection) {
+				const wrapperElement = selection.root.find(
+					'[data-cke-widget-wrapper]'
+				);
+
+				if (wrapperElement) {
+					const elementList = wrapperElement.$;
+					if (elementList.length > 0) {
+						const lastElement = new CKEDITOR.dom.element(
+							elementList[elementList.length - 1]
+						);
+
+						const imageElement = lastElement.findOne('img');
+						const widgetElement = lastElement.findOne(
+							'[data-widget="videoembed"]'
+						);
+
+						if (imageElement && widgetElement) {
+							const range = editor.createRange();
+
+							range.setStart(widgetElement, 0);
+							range.setEnd(imageElement, 1);
+
+							selection.selectRanges([range]);
+							selection.selectElement(lastElement);
+						}
+					}
+				}
+			}
+		}, 0);
+	};
+
+	const EMBED_VIDEO_WIDTH = 560;
+	const EMBED_VIDEO_HEIGHT = 315;
+
+	const embedProviders = [
```

Comment:
@csierra This are the templates we will save when a video is added to the editor. Please configure Antisamy to accept this tags and attributes, thanks!